lol_alloc/src/free_list_allocator.rs

```diff
@@ -173,14 +173,13 @@ fn full_size(layout: Layout) -> usize {
 // From https://github.com/wackywendell/basicalloc/blob/0ad35d6308f70996f5a29b75381917f4cbfd9aef/src/allocators.rs
 // Round up value to the nearest multiple of increment
 fn round_up(value: usize, increment: usize) -> usize {
```

Comment:
Looks like I forgot unit tests for this one. Some tests in the style of those for multiple_below would be great.

Reply:
Done.